pkg/commands/logging/datadog/create.go

```diff
@@ -63,9 +64,9 @@ func NewCreateCommand(parent cmd.Registerer, globals *config.Data, data manifest
 	})
 	c.CmdClause.Flag("region", "The region that log data will be sent to. One of US, US3, US5, or EU. Defaults to US if undefined").Action(c.Region.Set).StringVar(&c.Region.Value)
 	c.CmdClause.Flag("format", "Apache style log formatting. For details on the default value refer to the documentation (https://developer.fastly.com/reference/api/logging/datadog/)").Action(c.Format.Set).StringVar(&c.Format.Value)
```

Comment:
I would say for flags like this where the description is very specific, we either...

A.) modify the function to accept a string that can be 'appended' if provided.

or

B.) Just remove the difference (e.g. remove "For details on the default value...").

The latter might be better as some flag differences might not be as simple as simply _appending_ a value to the flag description. Also, it's not that useful providing such a long link in the description (IMHO).